src/article-view.js

```diff
@@ -0,0 +1,16 @@
+(function(exports) {
+  function ArticleView(articleJSON, articleModel) {
```

Comment:
Does this need to take both `articleJSON` and `articleModel`? Doesn't the articleModel already contain the json?